packages/protocol/contracts/common/EssentialContract.sol

```diff
@@ -21,11 +21,11 @@ abstract contract EssentialContract is UUPSUpgradeable, Ownable2StepUpgradeable,
     bytes32 private constant _REENTRY_SLOT =
         0xa5054f728453d3dbe953bdc43e4d0cb97e662ea32d7958190f3dc2da31d9721b;
 
-    /// @dev Slot 1.
-    uint8 private __reentry;
-
-    uint8 private __paused;
-
+    /// @dev Slot 1&2 - not optimized if we store them in 1 slot, because they neither read nor
+    /// written at the same time.
+    uint256 private __reentry;
```

Comment:
Dani, please revert this change as it will require us to reset the current testnet.
We are not optimizing, but it's still OK to have them use 1 slot.

Reply:
Ok, will do !